redshells/model/graph_convolutional_matrix_completion.py

```diff
@@ -213,7 +175,129 @@ def _to_constant(x):
         return tf.constant(x) if x is not None else None
 
 
-class GraphConvolutionalMatrixCompletion(object):
+class GraphConvolutionalMatrixCompletionGraph(GraphConvolutionalMatrixCompletionGraphCore):
```

Comment:
This class name seems to be too long. It may be a good chance to reconsideration for the class architecture.

Reply:
I've refactored the Graph class architecture.